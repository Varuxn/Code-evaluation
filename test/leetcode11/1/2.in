96 190
34 70 779
39 45 702
0 1 715
60 88 817
19 25 280
27 4 149
41 42 902
5 11 905
14 71 224
8 22 300
10 27 505
85 86 370
82 85 783
78 26 427
56 65 440
42 60 558
18 76 393
85 92 462
19 20 49
12 7 918
1 3 546
6 41 34
89 11 573
73 81 316
15 95 67
5 53 543
14 17 133
62 71 307
93 31 197
29 56 151
45 76 408
51 52 23
91 4 314
76 0 556
26 33 263
26 30 340
26 43 925
20 63 668
39 64 667
87 82 26
7 47 575
94 19 405
43 57 676
47 59 867
53 95 698
17 23 43
6 8 469
66 64 266
27 28 111
3 68 935
74 91 341
29 44 274
37 60 591
87 16 919
62 51 225
87 70 950
44 67 813
0 2 146
53 54 256
29 74 590
49 86 923
33 14 339
42 87 602
4 9 158
25 72 434
52 43 150
55 34 296
77 60 310
14 53 942
4 21 471
49 30 863
18 31 332
24 19 31
22 28 967
6 12 157
24 35 349
72 73 269
13 85 87
73 8 852
3 36 175
52 72 899
43 27 300
44 84 318
45 6 990
30 4 726
54 61 373
27 93 845
48 79 415
85 81 158
49 80 261
48 62 996
35 48 42
26 0 637
80 25 183
10 16 309
64 51 242
30 52 759
79 45 316
54 66 43
31 55 281
3 25 643
5 14 435
24 93 282
20 34 455
29 37 970
20 56 765
93 41 577
63 71 39
40 59 720
95 51 565
27 37 259
14 28 894
4 57 185
46 78 670
72 95 905
4 5 48
29 94 660
22 36 438
26 52 725
13 18 525
53 91 117
74 61 635
9 33 359
8 15 297
14 52 204
30 64 465
42 46 600
14 40 289
20 52 323
9 49 546
58 85 534
2 4 143
44 75 385
20 51 295
83 74 511
8 38 522
13 26 146
10 62 607
75 35 415
59 50 725
66 46 765
27 18 684
19 46 260
8 24 889
20 55 254
26 69 60
51 45 57
17 77 241
17 83 188
68 90 513
30 43 467
22 25 545
1 6 447
4 39 329
37 58 745
5 7 535
34 74 211
59 24 549
10 70 446
77 8 21
67 73 352
53 55 473
61 82 549
69 36 504
6 9 101
51 35 4
24 32 401
52 21 776
3 10 753
59 89 655
47 95 584
63 49 489
69 75 615
75 58 127
62 54 84
13 23 541
13 1 392
49 77 447
91 82 591
57 78 554
6 19 822
26 29 898
89 24 448
0 50 185
0 40 642
15 64 56
11 13 441
30 94 349
36 40 176
55 46 28
