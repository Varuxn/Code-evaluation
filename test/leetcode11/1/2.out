3 4 5 8 9 10 11 13 14 15 16 17 18 21 23 24 26 28 29 31 34 35 39 40 45 48 49 51 54 57 58 61 63 65 66 67 69 71 72 74 76 77 79 81 82 85 88 89 91 93 94 95 97 98 99 107 110 112 115 120 124 127 131 132 133 135 136 138 144 145 146 147 148 149 153 155 156 157 158 159 160 162 164 165 166 173 174 176 182 183 185 186 187 188 189 

